### PR TITLE
Missing image

### PR DESCRIPTION
**Pull request by chuenlok**
This is based on the test db PR. 
1. `'type'+missing_card_image` would be used for missing image for the `type.
   For example, if`Henry Tai+image`does not exist, it will take the image from`user+missing_card_image`.
2. We are having `+logo` in metric pages, but user's images are on `+image`. It would render `+image` for `user+logo` with the params.

### CODE REVIEW

**Comment by ethn** [comment]
would you be willing to re-create this PR so I can see the changed files?  (the diff shows all the test db stuff)

**Comment by chuenlok** [comment]
I thoughtmerging the another PR will make it look normal. I merged from wagn/master again and it looks ok now

File: mod/wikirate/set/right/image.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+format :html do
+
+  view :missing do |args|
+    # try to get type+missing_image_card's image to display
+    # EX: user+missing_image_card
+    parent_type_name = Card[card.left.type_id].name
+    missing_image_card = Card["#{parent_type_name}+missing_image_card"]
```
Comment (ethn) [inline mod/wikirate/set/right/image.rb]: needs a codename

File: mod/wikirate/set/type_plus_right/user/wikirate_logo.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+format :html do 
+  view :missing do |args|
+    username = card.cardname.left
+    # user's image is in +image but in metric page, we are using +logo
+    # a virtual card with structure won't work as it cannot have the args from +logo
+    if user_image = Card["#{username}+image"]
```
Comment (ethn) [inline mod/wikirate/set/type_plus_right/user/wikirate_logo.rb]: codename

**Pull request merged**